### ISSUE TITLE
doc: Write a how-to for using processors

### ISSUE THREAD

**Issue by hariso** [ConduitIO/conduit-site#187]

**Comment by raulb** [comment]
@hariso is this what you had in mind or something else? https://github.com/ConduitIO/conduit-site/pull/165

**Comment by hariso** [comment]
@raulb Not really, it's more about how to use existing processors than about writing new processors. i.e. it's more like "how to add a processor to a pipeline" where we explain how to use a processor in a pipeline.
